2.train-VAE/3.beta_vae_training_extract_model_components.ipynb

```diff
@@ -9,7 +9,7 @@
      "name": "stderr",
```

Comment:
<p>Please see comments left in <code>2.train-VAE/2.fetch_hyper_params.ipynb</code> file about hardcoding values and the possible use of <code>yaml</code> or <code>json</code>    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/gene_dependency_representations/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>